solutions/devsprint-leonardo-santos-3/FinanceAppTests/Components/TabBarControllerTests.swift

```diff
@@ -0,0 +1,31 @@
+import XCTest
+@testable import FinanceApp
+
+class TabBarControllerTests: XCTestCase {
+    
+    private let sut = TabBarController()
+    
+    func test_numberOfViewControllersOnTabBar_shouldBeTwo(){
+        sut.viewWillAppear(false)
+        
+        XCTAssertEqual(sut.viewControllers?.count, 2)
+        XCTAssertEqual(sut.tabBar.items?.count, 2)
+    }
+    
+    func test_firstViewControllerType_shouldBeHomeViewController(){
+        sut.viewWillAppear(false)
+        
+        sut.viewControllers?[0].children.forEach({ (element) in
+            XCTAssertTrue(type(of: element) == HomeViewController.self, "Element type is: \(String(describing: type(of: element))) " )
```

Comment:
Ao invés de verificar os elementos de UI, optei por verificar se as controllers que estão dentro do array são dos tipos esperados pelo app. Creio que essa verificação cabe em um teste de UI mesmo.
E se caso isso for válido, não achei uma forma melhor de comparar os tipos que ficam dentro do array das viewControllers.